crates/actor/src/compat/wasmcloud/bus/host.rs

```diff
@@ -60,30 +60,9 @@ fn host_call(
 }
 
 pub fn call_sync(
-    target: Option<&TargetEntity>,
+    target: Option<&WrpcTarget>,
     operation: &str,
     payload: &[u8],
 ) -> Result<Vec<u8>, String> {
-    match target {
-        None => {
-            let (namespace, operation) = operation
-                .rsplit_once('/')
-                .ok_or_else(|| "invalid operation format".to_string())?;
-            host_call("", namespace, operation, payload)
-        }
-        Some(TargetEntity::Link(binding)) => {
-            let (namespace, operation) = operation
-                .rsplit_once('/')
-                .ok_or_else(|| "invalid operation format".to_string())?;
-            host_call(
-                binding.as_deref().unwrap_or_default(),
-                namespace,
-                operation,
-                payload,
-            )
-        }
-        Some(TargetEntity::Actor(
-            ActorIdentifier::PublicKey(namespace) | ActorIdentifier::Alias(namespace),
-        )) => host_call("", namespace, operation, payload),
-    }
+    panic!("not supported, only wrpc")
```

Comment:
😱 

Real talk though, I think a `bail` would be nice to have here just in case

Reply:
I went and removed `compat` (and the feature flags for wasm modules)...